providers/linenotify/src/lib/linenotify.provider.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { LinenotifyProvider } from './linenotify.provider';
```

Comment:
```suggestion
import { LinenotifyProvider } from './line-notify.provider';
```
And rename the file, please.